Lib/http/server.py

```diff
@@ -970,7 +991,7 @@ def _get_best_family(*address):
 def test(HandlerClass=BaseHTTPRequestHandler,
          ServerClass=ThreadingHTTPServer,
          protocol="HTTP/1.0", port=8000, bind=None,
-         tls_cert=None, tls_key=None, tls_password=None):
+         tls_cert=None, tls_key=None, tls_password=None, response_headers=None):
```

Comment:
```suggestion
         tls_cert=None, tls_key=None, tls_password=None,
         response_headers=None):
```

Let's group the parameters per purpose